src/libraries/System.Private.CoreLib/src/System/WeakReference.cs

```diff
@@ -9,7 +9,7 @@ namespace System
     [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public partial class WeakReference : ISerializable
     {
-        // If you fix bugs here, please fix them in WeakReference<T> at the same time.
+        // If you fix bugs here, please fix them in WeakReference<T> at the same time
```

Comment:
Nit: random comment change appears unrelated?

Reply:
It is unrelated.  I changed a comment to make CI run the libraries subset of the runtime pipeline which includes the windows 7 leg.  I can remove the change before merging.